src/main.c

```diff
@@ -1415,6 +1415,7 @@ emulator_loop(void *param)
 
 		if (video_get_irq_out() || via1_irq() || (has_via2 && via2_irq())) {
 //			printf("IRQ!\n");
+			memory_set_rom_bank(0);
```

Comment:
Since there isn't any hardware yet that implements this behavior, I think we might want to put this behind a command line argument, default to off.